cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -393,6 +393,8 @@ spec:
               value: "https://71964476292e4087af8d5072afe43abd@o394722.ingest.sentry.io/5245431"
             {{- end }}
             {{- end }}
+            - name: OUT_OF_CLUSTER_ENABLED
```

Comment:
this env variable name needs to be a bit more specific, as noted below

Reply:
Updated.